templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -140,6 +130,16 @@ contents:
       "KILL", 
     ]
 
+    # List of default sysctls. If it is empty or commented out, only the sysctls
+    # defined in the container json file by the user/kube will be added.
+    default_sysctls = []
+
+    # List of additional devices. specified as
+    # "<device-on-host>:<device-on-container>:<permissions>", for example: "--device=/dev/sdc:/dev/xvdc:rwm".
+    #If it is empty or commented out, only the devices
+    # defined in the container json file by the user/kube will be added.
+    additional_devices = []
+
     # hooks_dir_path is the oci hooks directory for automatically executed hooks
     hooks_dir_path = "/usr/share/containers/oci/hooks.d"
```

Comment:
it seems hooks_dir_path has been replaced with hooks_dir. hooks_dir is a slice so we will want:
`hooks_dir = ["/usr/share/containers/oci/hooks.d", ]`

Reply:
Again if we can just comment this out and not have to change the MCO when crio changes...